### PR TITLE
Feature helpers

### PR DESCRIPTION
**Pull request by krishnajainGKMIT**

### CODE REVIEW

File: ] [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Merge branch 'feature-already-registered-test' of https://github.com/krishnajainGKMIT/practice-tdd into feature-helpers
+# Please enter a commit message to explain why this merge is necessary,
+# especially if it merges an updated upstream into a topic branch.
```
Comment (paras2103) [inline ]]: what is this file?

File: helpers/errorHelp.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+const errorHelper = (code, name, message, actual) => {
+  if (code == 409 || 422) {
+    type = "service error ";
+  } else if (code == 400) {
+    type = "client error";
```
Comment (paras2103) [inline helpers/errorHelp.js]: file name is wrong, 
errors.helper.js

File: helpers/passord.helper.js
Comment (paras2103) [inline helpers/passord.helper.js:1]: password spelling is wrong

File: ] (hunk 1 above)
Comment (krishnajainGKMIT) [reply ]]: file  removed

File: helpers/errorHelp.js (hunk 2 above)
Comment (krishnajainGKMIT) [reply helpers/errorHelp.js]: changes implemented

File: helpers/passord.helper.js
Comment (krishnajainGKMIT) [reply helpers/passord.helper.js:1]: chanegs implemented

**Pull request merged**